### PR TITLE
update ci

### PR DESCRIPTION
**Pull request by luward**

### CODE REVIEW

File: environment.yml [hunk 1]
```diff
@@ -22,7 +22,7 @@ dependencies:
 - scipy
 - jupyterlab
 - matplotlib
-- seaborn
+- seaborn==0.10.1
```
Comment (amageh) [inline environment.yml:25]: Why is this needed?

**Review by amageh — APPROVED** [review]

File: environment.yml (hunk 1 above)
Comment (luward) [reply environment.yml:25]: The jointplot function inputs were changed in 0.11 and produce an error, I couldn't find the argument in question(stat_func=None) in the docs however, so I just set it to to 0.10.1

Comment (amageh) [reply environment.yml:25]: Okay thanks, I was just curious :)
